.gitignore

```diff
@@ -6,5 +6,11 @@
 *.iml
 target
 eclipse-target
+maven-archiver
+maven-status
+!/java/impl/src/test/resources/testdata/localmavenproject/maven.project/core/target
+!/java/impl/src/test/resources/testdata/localmavenproject/maven.project/core/eclipse-target
+!/java/impl/src/test/resources/testdata/localmavenproject/maven.project/api/target
+!/java/impl/src/test/resources/testdata/localmavenproject/maven.project/api/eclipse-target
```

Comment:
Adding binary class files to git is IMHO not an acceptable approach.
Please remove this lines and the added *.class files
Instead introduce a maven build via module (or if required via exec plugin) to mak tests work.